src/ragas/metrics/__init__.py

```diff
@@ -1,15 +1,15 @@
-from ragas.metrics.answer_correctness import AnswerCorrectness, answer_correctness
-from ragas.metrics.answer_relevance import AnswerRelevancy, answer_relevancy
-from ragas.metrics.answer_similarity import AnswerSimilarity, answer_similarity
-from ragas.metrics.context_precision import (
+from ragas.metrics._answer_correctness import AnswerCorrectness, answer_correctness
+from ragas.metrics._answer_relevance import AnswerRelevancy, answer_relevancy
+from ragas.metrics._answer_similarity import AnswerSimilarity, answer_similarity
+from ragas.metrics._contexts import (
     ContextPrecision,
     ContextRelevancy,
     context_precision,
     context_relevancy,
 )
```

Comment:
@tinomaxthayil and @shahules786 should we split this into 2 files?

Reply:
you meant Class and instance for _contexts?

Reply:
@jjmachan Not sure what is ideal here, since both are metrics to evaluate contexts, I would suggest to keep as it is.